Observing-Interface-Setup/efd-kafka.rst

```diff
@@ -0,0 +1,89 @@
+#############
+EFD SAL Kafka 
+#############
+
+Overview
+^^^^^^^^
+This topic covers information on how to perform actions on the EFD SAL Kafka producers/writers on the non-main domains.
+The summit kafka producers are currently running on the azar2 node at azar2.cp.lsst.org.
+So all of these sub-procedures will be run on azar2.
+
+.. warning:: All of these procedures have the potential to inflict serious instablity on the summit stream of data to the EFD. These procedures must be executed with the upmost caution and in some cases should be used as a last resort.
+
```

Comment:
I think the issue here is that you say it's highly dangerous and only do it as a last resort, but what are the other things that should be done first?